src/article/article-furniture.ts

```diff
@@ -1,20 +1,28 @@
 export const articleFurniture = (doi: string, noHeader: boolean) => `<div class="article-furniture">
```

Comment:
optional, as you found it in a bad state, but could we sort the line indentation on this string?